reana/cli.py

```diff
@@ -594,6 +601,9 @@ def git_status(component):  # noqa: D301
             click.secho(' @ {0} {1}'.format(branch, commit))
         else:
             click.secho(' @ {0} {1}'.format(branch, commit), fg='red')
+        if short:
+            cmd = 'git status -s'
```

Comment:
I would maybe use the more verbose mode `git status --short` so it is more descriptive and easy to read.